File: JavaPatternPrograms/Symbol Patterns/Pattern 46.java
 public class Pattern_132
 {
     public static void main(String args[])
     {
      int i;
      int j;
 
      int n = 8; //size (even)
 
      int c1 = (n - 1) / 2; //cond
 
      int c2 = 3 * n / 2 - 1; //cond
 
      for (i = 0;i < n;i++)
      {
       for (j = 0;j < n;j++)
       {
         if (i + j <= c1  || i - j >= c1  || j - i >= c1  || i + j 
       >= c2)
         {
          System.out.print("* ");
         }
         else
         {
          System.out.print("  ");
         }
       }
       System.out.println("");
      }
     }
 }

File: PythonPatternPrograms/SymbolPatterns/Pattern 46.py
n = 8  # size even
c1 = (n - 1) // 2  # cond
c2 = 3 * n // 2 - 1  # cond

for x in range(0, n):
    for y in range(0, n):
        if (x + y <= c1 or x - y >= c1 or y - x >= c1 or x + y >= c2):
            print("* ", end="")  # *space
        else:
            print("  ", end="")  # two spaces
    print()